<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingDialog</class>
 <widget class="QDialog" name="DatabaseSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>140</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Create new Catalog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="selectFolderButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;SelectFolder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pathEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string extracomment="A message indicating that one catalog can register multiple subfolders at once">Each catalog can be created including Subfolders and/or Archives</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatabaseSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatabaseSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DatabaseSettingDialog</receiver>
   <slot>onNameEdit_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DatabaseSettingDialog</receiver>
   <slot>onPathEdit_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>DatabaseSettingDialog</receiver>
   <slot>onSelectFolderButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNameEdit_textChanged(QString)</slot>
  <slot>onPathEdit_textChanged(QString)</slot>
  <slot>onSelectFolderButton_clicked()</slot>
 </slots>
</ui>
